### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by IvanIvanoff**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/mandrill_api.ex [hunk 1]
```diff
@@ -1,5 +1,6 @@
 defmodule Sanbase.MandrillApi do
   require Sanbase.Utils.Config, as: Config
+  require Application
```
Comment (tspenov) [inline lib/sanbase/mandrill_api.ex:3]: it should work without this require, right ?

File: lib/sanbase/accounts/email_jobs.ex [hunk 2]
```diff
@@ -72,7 +72,6 @@ defmodule Sanbase.Accounts.EmailJobs do
         :first_education_email -> days_after(4)
         :trial_suggestion -> days_after(6)
         :second_education_email -> days_after(7)
-        _ -> nil
```
Comment (tspenov) [inline lib/sanbase/accounts/email_jobs.ex:75]: Here the default is `nil` which means don't schedule for later but send immediately after insert. I don't see a reason for removing it but also it is not reached now, so I am ok both ways

**Review by tspenov — APPROVED** [review]

File: lib/sanbase/accounts/email_jobs.ex (hunk 2 above)
Comment (IvanIvanoff) [reply lib/sanbase/accounts/email_jobs.ex:75]: It's not used and emits warnings. But my bigger issue is that such catch all cases can bite us back in the future - one might forget to specify when an email must be scheduled or mistypes the name - both these would result in an error if we remove it.

File: lib/sanbase/mandrill_api.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase/mandrill_api.ex:3]: It emits warnings with the latest elixir version. I think it will stop working in some of the next versions

**Pull request merged**